connect/runtime/src/test/java/org/apache/kafka/connect/runtime/LoggersTest.java

```diff
@@ -41,12 +45,19 @@ public class LoggersTest {
     private static final long INITIAL_TIME = 1696951712135L;
     private Loggers.Log4jLoggers loggers;
     private Time time;
+    Level originalRootLevel;
 
     @BeforeEach
     public void setup() {
+        originalRootLevel = LogManager.getRootLogger().getLevel();
         time = new MockTime(0, INITIAL_TIME, 0);
         loggers = (Loggers.Log4jLoggers) Loggers.newInstance(time);
     }
+    
+    @AfterEach
+    public void teardown() {
+        Configurator.setAllLevels(LogManager.ROOT_LOGGER_NAME, originalRootLevel);
```

Comment:
This changes all 'child' loggers, right? If so, it is not a kind of restoring logger levels